Rubberduck.Resources/UnitTesting/TestExplorer.de.resx

```diff
@@ -280,7 +280,7 @@
     <value>SpaetesBinden</value>
   </data>
   <data name="UnitTest_NewModule_LateBindDirective" xml:space="preserve">
-    <value>BindeTestsSpaet</value>
+    <value>SpaetesBinden Tests</value>
```

Comment:
Bin unsicher, wo das verwendet wird. Sind Leerzeichen dort Sinnvoll? Die Vermeidung des Umlautes impliziert eine Verwendung, bei der Leerzeichen gefährlich sein könnten...

Reply:
This should definitely not contain a space. It is used as the name of a compilation argument in a precompiler directive.
I would actually say that we should not translate this at all since it becomes part of the code. For now, I will revert this change as part of my cleanup commit for this PR.